File: Model/Entities/Mau.cs
namespace Model.Entities
{
    public class Mau
    {
        public int Id { get; set; }
        public string DuongDanAnh { get; set; }
        public string DoPhanGiai { get; set; }
        public List<KhungNhanDang> DsBien { get; set; }
    }
}

File: server5/common-model/src/main/java/com/trafficsigndetector/sharedmodel/Mau.java
package com.trafficsigndetector.sharedmodel;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
public record Mau(
        @JsonProperty("Id") Integer id,
        @JsonProperty("DuongDanAnh") String duongDanAnh,
        @JsonProperty("DoPhanGiai") String doPhanGiai,
        @JsonProperty("DsBien") List<KhungNhanDang> dsBien
) {
}
